src/core_atmosphere/Registry.xml

```diff
@@ -1812,6 +1812,13 @@
                 <var name="surface_pressure" type="real" dimensions="nCells Time" units="Pa"
                      description="Diagnosed surface pressure"/>
 
+                <!-- observation-related variables interfacing to Non-Variational SAtellite-based Cloud Analysis (SACA)  -->
+                <var name="cldmask" type="real" dimensions="nCells Time" units="unitless"
```

Comment:
I am interested to see the data relationships to know why `cldmask` is needed and why `cldfrac` is not enough.  However I accept that it is needed for now.

Should these new variables get packaged under a new namelist option similar to our DA cycling variables?

Reply:
`cldmask` and `brtemp` provide the information on the observation. They modify the (model) `cldfrac` values (and eventually `qc`, `qi`, `qs`, and `qv`), for example, making `cldfrac=0.0` where `cldmask=0` (clear).

These two variables are added to the `jedi_da` package in [522d715](https://github.com/JCSDA-internal/MPAS-Model/pull/43/commits/522d7151f4d2f0f50090e38aaec7fb2cf2a4905a) , as suggested.

Reply:
Is `cldmask` read/written from/to file?  Or used in the MPAS-Model source code?

If either is yes, the `cldmask` is needed.  I think that if neither answer is yes, then some placeholder array or MPAS field instance can be used instead.

Reply:
`cldmask` is read from file, but it is not used in the MPAS-Model source code.

Same for `brtemp`.